src/Providers/Composer/ComposerServiceProvider.php

```diff
@@ -27,12 +27,12 @@ final class ComposerServiceProvider extends BaseServiceProvider
     public function register(): void
     {
         $this->app->singleton(
-            'composer',
+            'composer_zero',
```

Comment:
laravel migration commands now uses laravel's composer version for `dumpAutoloads`, and laravel zero's version was conflicting the name, and as we don't use this name i just changed... it may be a breaking change... not sure 🤔